packages/mui-codemod/src/v6.0.0/sx-prop/test-cases/basic-sx.expected.js

```diff
@@ -0,0 +1,46 @@
+<Toolbar
+  variant="regular"
+  sx={(theme) => ({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    flexShrink: 0,
+    borderRadius: '999px',
+    bgcolor: 'hsla(220, 0%, 0%, 0.7)',
+    backdropFilter: 'blur(24px)',
+    maxHeight: 40,
+    border: '1px solid',
+    borderColor: 'divider',
+    boxShadow:
+      '0 1px 2px hsla(210, 0%, 0%, 0.5), 0 2px 12px hsla(210, 100%, 25%, 0.3)',
+    ...theme.applyStyles("light", {
```

Comment:
Should we default to adding the dark styles separately?

Reply:
I don't think we should do that. Codemod should not target specific color-scheme. Currently, the transformation already matches the intent of the developers.